kairon/shared/analytics/analytics_pipeline_processor.py

```
@@ -0,0 +1,152 @@
+            settings = AnalyticsPipelineConfig.objects(id=event_id, bot=bot, status=True).get()
+
+            settings.pipeline_name = config["pipeline_name"]
+            settings.callback_name = config.get("callback_name")
+
+            scheduler_config = SchedulerConfiguration(**config["scheduler_config"])
+
+            if scheduler_config.expression_type == "epoch":
+                try:
+                    epoch_time = int(scheduler_config.schedule)
+                except ValueError:
+                    raise AppException("schedule must be a valid integer epoch timestamp")
```

Ruff (0.14.7)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)